contracts/src/contracts/utils.py

```diff
@@ -435,7 +443,7 @@ def setupAuctionApp(
         # additional min balance to opt into NFT
         + 100_000
         # 3 * min txn fee
-        + 3 * 1_000
+        + 4 * 1_000
```

Comment:
Was this changed to 4 for testing, or is intentional? If so, I presume the comment above should change to 4 as well?